src/background/analysis/constants.js

```diff
@@ -10,8 +10,18 @@ export const MAX_BYTE_LEN = 200000
  */
 export const MAX_CHAR_LEN = 100000
 
+/**
+ * Used in setTimeout when changing favicon
+ */
+export const FIVE_SEC_IN_MILLIS = 5000
+
+/**
+ * Used to see if evidence is large for a first party
```

Comment:
Probably mention that this is used to make a decision about the rocketship svg here.